SmartDeviceLink/private/SDLLifecycleManager.m

```diff
@@ -395,6 +395,8 @@ - (void)didEnterStateConnected {
 
         weakSelf.registerResponse = (SDLRegisterAppInterfaceResponse *)response;
         [SDLGlobals sharedGlobals].rpcVersion = [SDLVersion versionWithSDLMsgVersion:weakSelf.registerResponse.sdlMsgVersion];
+        SDLLogD(@"Did register app; RPC version: %@, SDL version: %@, starting languages: (VR) %@, (HMI) %@, vehicle type: %@", weakSelf.registerResponse.sdlMsgVersion, weakSelf.registerResponse.sdlVersion, weakSelf.registerResponse.language, weakSelf.registerResponse.hmiDisplayLanguage, weakSelf.registerResponse.vehicleType);
```

Comment:
When connecting to SYNC 3 I get **Did register app; RPC version: 4.2.(null)...**. Is it possible not to print the patch version if it is `null`?

Reply:
Fixed